test/Microsoft.NET.Build.Tests/WorkloadTests.cs

```diff
@@ -259,7 +259,7 @@ public void It_should_get_suggested_workload_by_GetRequiredWorkloads_target()
                 .BeEquivalentTo("android");
         }
 
-        [Theory(Skip = "https://github.com/dotnet/installer/issues/13361")]
+        [Theory]
```

Comment:
Unskipped this test failed. Details here: https://github.com/dotnet/installer/issues/13361#issuecomment-2183493550